vulnerabilities/tests/test_pysec.py

```diff
@@ -0,0 +1,292 @@
+# Copyright (c) nexB Inc. and others. All rights reserved.
+# http://nexb.com and https://github.com/nexB/vulnerablecode/
+# The VulnerableCode software is licensed under the Apache License version 2.0.
+# Data generated with VulnerableCode require an acknowledgment.
+#
+# You may not use this software except in compliance with the License.
+# You may obtain a copy of the License at: http://apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+#
+# When you publish or redistribute any data created with VulnerableCode or any VulnerableCode
+# derivative work, you must accompany this data with the following acknowledgment:
+#
+#  Generated with VulnerableCode and provided on an "AS IS" BASIS, WITHOUT WARRANTIES
+#  OR CONDITIONS OF ANY KIND, either express or implied. No content created from
+#  VulnerableCode should be considered or used as legal advice. Consult an Attorney
+#  for any legal advice.
+#  VulnerableCode is a free software tool from nexB Inc. and others.
+#  Visit https://github.com/nexB/vulnerablecode/ for support and download.
+import datetime
+import json
+import os
+from unittest import TestCase
+
+from packageurl import PackageURL
+from univers.version_range import PypiVersionRange
+from univers.versions import PypiVersion
+from univers.versions import SemverVersion
+
+from vulnerabilities.importer import AdvisoryData
+from vulnerabilities.importer import AffectedPackage
+from vulnerabilities.importer import Reference
+from vulnerabilities.importers.pysec import parse_advisory_data
+
+BASE_DIR = os.path.dirname(os.path.abspath(__file__))
+TEST_DATA = os.path.join(BASE_DIR, "test_data/pysec", "pysec_test.json")
+
+
+class TestPyPLImporter(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        with open(TEST_DATA) as f:
+            cls.mock_response = json.load(f)
+
+    def test_to_advisories(self):
+        expected_advisories = [
+            AdvisoryData(
+                aliases=["CVE-2021-40831"],
+                summary="Improper certificate management in AWS IoT Device SDK v2",
+                affected_packages=[
+                    AffectedPackage(
+                        package=PackageURL(type="PyPI", name="awsiotsdk"),
+                        affected_version_range=PypiVersionRange(
+                            [
```

Comment:
For readability, please create a variable with this long list outside before using it here.